Practicing English.txt

```diff
@@ -0,0 +1,15 @@
+High priority
+-------------
+Harry potter
+
```

Comment:
[Ben-Naim A. Entropy Demystified - The Second Law Reduced to Plain Common Sense.pdf](https://github.com/Sumytech/reading-list/files/2223158/Ben-Naim.A.Entropy.Demystified.-.The.Second.Law.Reduced.to.Plain.Common.Sense.pdf)

Reply:
I'd like to read it first,